src/NuGetPackageVerifier/CompositeRules/SigningVerificationCompositeRule.cs

```diff
@@ -15,6 +15,7 @@ public class SigningVerificationCompositeRule : IPackageVerifierRule
             new AuthenticodeSigningRule(),
             new PowerShellScriptIsSignedRule(),
             new RequiredNuSpecInfoRule(),
+            new DefaultCompositeRule(),
```

Comment:
We missed verification in BrowserLink because the file was missing. I thought I'd be nice to run strong name verify rule and serviceable rule on these binaries again just as a secondary check ?